gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherSettings.java

```diff
@@ -123,17 +122,17 @@
         RetryParams.newBuilder()
             .setRetryBackoff(
                 BackoffParams.newBuilder()
-                    .setInitialDelayMillis(100L)
+                    .setInitialDelay(Duration.millis(100L))
                     .setDelayMultiplier(1.2)
-                    .setMaxDelayMillis(1000L)
+                    .setMaxDelay(Duration.millis(1000L))
                     .build())
             .setTimeoutBackoff(
                 BackoffParams.newBuilder()
-                    .setInitialDelayMillis(300L)
-                    .setDelayMultiplier(1.3)
-                    .setMaxDelayMillis(3000L)
+                    .setInitialDelay(Duration.millis(2000L))
+                    .setDelayMultiplier(1.5)
+                    .setMaxDelay(Duration.millis(30000L))
                     .build())
-            .setTotalTimeout(30000L)
+            .setTotalTimeout(Duration.millis(45000L))
```

Comment:
1. In general I prefer to avoid making client of API X to be familiar with API Y (that X is using). Internal usage is totally fine.

2. Java 8 has supports for Duration (which will remove the need to use Joda). I didn't check if their semantics is identical or very close.

For these 2 reasons we chose to use this signature instead (int time, TimeUnit unit). (`TimeUnit` is part of the Java SDK). Later we can add (or replace) with Java 8 specifics.

Fine as is, wanted to voice the reasoning and style we use.